CameraOverlay/commons.py

```diff
@@ -1,47 +1,71 @@
 import os
 import json
+import random
+import fnmatch
 from dotenv import load_dotenv
 
-from .topics import *
```

Comment:
Cheers. This was causing the script to not work.

Reply:
i did test if before pushing. and it still does work on my laptop, strange...

Reply:
how are you trying to run it?

Reply:
`python3 orchestrator.py` ?

Reply:
I removed the . in front of the topics and it worked again.

Reply:
oh right, this _was_ causing it to not work. i thought it _is_ causing it to not work. there's no issue at the moment right

Reply:
Nope! This is a good change!